README.md

```diff
@@ -31,22 +32,28 @@
 
 1. On top of the fields already available in AB3, the user will be able to input the following:
```

Comment:
Perhaps the `tag` part could be here? "On top of the fields already available in AB3, excluding the `tag` feature, the user will be able to input the following:...". Just a suggestion!

Reply:
Changed thanks